tests/commands/test_shm.py

```diff
@@ -41,10 +43,12 @@ def test_infrastructure_auth_failure(
 class TestTeardownSHM:
     def test_teardown(
         self,
+        mocker,
         runner,
         mock_imperative_shm_teardown_then_exit,  # noqa: ARG002
         mock_shm_config_from_remote,  # noqa: ARG002
     ):
+        mocker.patch.object(Confirm, "ask", return_value="yes")
```

Comment:
We can use the runner to [test input](https://typer.tiangolo.com/tutorial/testing/#call-pytest).